frontend/public/components/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -91,35 +96,51 @@ export const DetailsCard_ = connect(mapStateToProps)(
       <DashboardCard>
         <DashboardCardHeader>
           <DashboardCardTitle>Details</DashboardCardTitle>
+          <DashboardCardLink to="/settings/cluster/">View settings</DashboardCardLink>
         </DashboardCardHeader>
         <DashboardCardBody isLoading={flagPending(openshiftFlag)}>
           <DetailsBody>
             {openshiftFlag ? (
               <>
                 <DetailItem
-                  key="clusterid"
+                  title="Cluster API address"
```

Comment:
I don't think we're ready to switch to sentence case yet since it's inconsistent with every detail page. I know it's in @andybraren's mocks, but it's inconsistent with current console code.

Reply:
@spadgett Do you think we'll switch to sentence case Console-wide for 4.3, or should all of our 4.3-targeted mockups from now on use Title case instead?